stk/molecular/topologies/base.py

```diff
@@ -198,21 +204,22 @@ def __init__(self, del_atoms=True, track_fgs=True):
         self.del_atoms = del_atoms
         self.track_fgs = track_fgs
 
-    def build(self, macro_mol, bb_conformers=None):
+    def construct(self, mol, bb_conformers=None):
         """
-        Assembles ``rdkit`` instances of macromolecules.
+        Constructs :mod:`rdkit` instances of molecules.
 
-        This method places an ``rdkit`` molecule of the assembled
-        macromolecule into the :attr:`~.Molecule.mol` attribute of
-        :class:`.MacroMolecule`. It also creates the
-        :attr:`.MacroMolecule.bb_counter` and
-        :attr:`.MacroMolecule.bonds_made` attributes.
+        This method places an :mod:`rdkit` molecule of the
+        :class:`.ConstructedMolecule`
+        into the :attr:`~.Molecule.mol` attribute of
+        :class:`.ConstructedMolecule`. It also creates the
+        :attr:`.ConstructedMolecule.bb_counter` and
+        :attr:`.ConstructedMolecule.bonds_made` attributes.
 
         Parameters
         ----------
-        macro_mol : :class:`.MacroMolecule`
-            The :class:`.MacroMolecule` instance which needs to be
-            built.
+        mol : :class:`.ConstructedMolecule`
```

Comment:
is the use of 'mol' here not a bit confusing considering 'mol' is used everywhere else for the RDKit mol?